### DISCUSSION TITLE
iOS crash  failed assertion `lsLightFrustumBounds.min.y < lsLightFrustumBounds.max.y' How to limit

### DISCUSSION THREAD

**Discussion by guoaobao** [google/filament#6207] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/17892094/196429663-f13e9575-a66c-4451-a15e-b9a3072eaf20.png)
![image](https://user-images.githubusercontent.com/17892094/196429774-e2ce3b7a-4e92-4dff-a319-587d180088cb.png)
Infinite playback will crash. How do you limit it ？

**Comment by bejado** [comment, marked as answer]
Which version of Filament are you using? We've fixed some shadow-related crashes recently.
